HISTORY.md

```diff
@@ -1,6 +1,11 @@
 # Rocksdb Change Log
 ## Unreleased
 
+### Public API changes
+* Add `rocksdb_writebatch_update_timestamps`, `rocksdb_writebatch_wi_update_timestamps` in C API.
+* Add `rocksdb_iter_refresh` in C API.
+* Add `rocksdb_writebatch_create_with_params`, `rocksdb_writebatch_wi_create_with_params` to create WB and WBWI with all options in C API
```

Comment:
We have a new way of adding release notes; see "unreleased_history/README.txt" after updating against main branch.

Although, it isn't strictly required to have release notes for new C APIs, so simply deleting the notes here would also work. I'd do it myself, but don't have access to edit this branch.

Reply:
Changed, please check